editor/writing-flow/index.js

```diff
@@ -78,4 +152,18 @@ class WritingFlow extends Component {
 	}
 }
 
-export default WritingFlow;
+export default connect(
+	( state ) => ( {
+		blocks: getBlockUids( state ),
+		selectionStart: getMultiSelectedBlocksStartUid( state ),
+		selectionEnd: getMultiSelectedBlocksEndUid( state ),
+		multiSelectedBlocks: getMultiSelectedBlocks( state ),
```

Comment:
Since this is only ever used here:

```js
const hasMultiSelection = multiSelectedBlocks.length > 1;
```

we can be more parsimonious in `mapState`:

```js
( state ) => ( {
  …,
  hasMultiSelection: getMultiSelectedBlocks( state ).length > 1,
```

Reply:
Yep, that makes sense.